target/linux/kirkwood/image/Makefile

```diff
@@ -105,47 +112,54 @@ endef
 TARGET_DEVICES += linksys_viper
 
 define Device/raidsonic_ib-nas62x0
-  DEVICE_TITLE := RaidSonic ICY BOX IB-NAS62x0
+  DEVICE_VENDOR := RaidSonic
+  DEVICE_MODEL := ICY BOX IB-NAS62x0
   DEVICE_DTS := kirkwood-ib62x0
   BOARD_NAME := ib62x0
   SUPPORTED_DEVICES += ib62x0
 endef
 TARGET_DEVICES += raidsonic_ib-nas62x0
 
 define Device/seagate_dockstar
+  DEVICE_VENDOR := Seagate
+  DEVICE_MODEL := FreeAgent Dockstar
   DEVICE_DTS := kirkwood-dockstar
-  DEVICE_TITLE := Seagate FreeAgent Dockstar
   BOARD_NAME := dockstar
   SUPPORTED_DEVICES += dockstar
 endef
 TARGET_DEVICES += seagate_dockstar
 
 define Device/seagate_goflexnet
-  DEVICE_TITLE := Seagate GoFlexNet
+  DEVICE_VENDOR := Seagate
+  DEVICE_MODEL := GoFlexNet
   DEVICE_DTS := kirkwood-goflexnet
   BOARD_NAME := goflexnet
   SUPPORTED_DEVICES += goflexnet
 endef
 TARGET_DEVICES += seagate_goflexnet
 
 define Device/seagate_goflexhome
-  DEVICE_TITLE := Seagate GoFlexHome
+  DEVICE_VENDOR := Seagate
+  DEVICE_MODEL := GoFlexHome
   DEVICE_DTS := kirkwood-goflexhome
   BOARD_NAME := goflexhome
   SUPPORTED_DEVICES += goflexhome
 endef
 TARGET_DEVICES += seagate_goflexhome
 
 define Device/zyxel_nsa310b
-  DEVICE_TITLE := ZyXEL NSA310b
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := NSA310b
   DEVICE_DTS := kirkwood-nsa310b
   DEVICE_PACKAGES := kmod-r8169 kmod-gpio-button-hotplug kmod-hwmon-lm85
   BOARD_NAME := nsa310b
 endef
 TARGET_DEVICES += zyxel_nsa310b
 
 define Device/zyxel_nsa325
-  DEVICE_TITLE := ZyXEL NSA325 (v1 and v2)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := NSA325
+  DEVICE_VARIANT := (v1 and v2)
```

Comment:
This is a bit ugly, maybe use two sections?

Reply:
`v1/v2` is ok?

Reply:
Could it be that the v1 / v2 is the number of slots? https://www.zyxel.com/support/SupportLandingSR.shtml?c=gb&l=en&kbid=M-01820&md=NSA325%20v2

In that case it should be the device model and two sections (imho).

Reply:
Well, if it's no revision and they are not different, I would think about just removing the "(v1 and v2)" instead of introducing two identical sections.

Reply:
v1 and v2 is just a different case, it's the same device with same board and same specifications in both versions.

https://www.computerbase.de/2013-10/zyxel-nsa325-v2-test/

Reply:
Ok, let's remove it then.

Reply:
I wrote v1 and v2 in there when I added support because this way is clear that it will work for both v1 and v2 of the NAS.

Do what you must to get it in line with this change, but don't delete the `v1/v2` please.

I also suppose there are other devices that are basically identical even with a different "revision", just deleting info is bad.

Reply:
Well, sounds like `DEVICE_VARIANT := v1/v2` would serve anyone ...

Reply:
If there are no other revisions than v1 and v2, I would just remove this. otherwise I would prefer "v1/v2" but @mwarning you will probably be one of the biggest users of this feature so you can decide.

Reply:
imho it is better to always state the device revision, even if there is only one.

If you remove the revision, the people will not know if this image really works for both (all) revisions, or if it was made for v1 only, and it will brick v2.

Reply:
I fully agree with @bobafetthotmail on this one

Reply:
ok, I changed "(v1 and v2)" to "v1/v2".